schemas/ipi/v1.json

```diff
@@ -0,0 +1,172 @@
+                "maxLength": 40,
+                "examples": ["SERIAL NO: 8585C000-001"]
+              },
+              "unit_of_sale": {
+                "type": "string",
+                "description": "The unit of sale for the item produced by the business.",
+                "minLength": 1,
+                "maxLength": 40,
+                "examples": ["EACH"]
+              },
+              "terms_of_sale_1": {
+                "type": "string",
+                "description": "A separate field of the terms of sale for the item produced by the business, the contents and order of this field can change dependent on the number of characters.",
+                "minLength": 1,
+                "maxLength": 40,
+                "examples": ["SUPPLIER REF: 123073"]
+              },
+              "terms_of_sale_2": {
+                "type": "string",
+                "description": "A separate field of the terms of sale for the item produced by the business, the contents and order of this field can change dependent on the number of characters.",
+                "maxLength": 40,
+                "examples": ["PAYMENT: 60 DAYS NET"]
+              },
+              "terms_of_sale_3": {
+                "type": "string",
+                "description": "A separate field of the terms of sale for the item produced by the business, the contents and order of this field can change dependent on the number of characters.",
+                "maxLength": 40,
+                "examples": ["CARRIAGE: FCA"]
+              },
+              "terms_of_sale_4": {
+                "type": "string",
+                "description": "A separate field of the terms of sale for the item produced by the business, the contents and order of this field can change dependent on the number of characters.",
+                "maxLength": 40,
+                "examples": ["CARRIAGE: FCA"]
+              },
+              "currency": {
+                "type": "string",
+                "description": "The currency for the item.",
+                "minLength": 1,
+                "maxLength": 25,
+                "examples": ["POUNDS STERLING"]
+              },
+              "country_of_origin": {
+                "type": "string",
+                "description": "The country of origin for the item",
+                "minLength": 1,
+                "maxLength": 25,
+                "examples": ["FRANCE"]
+              }
+            },
+            "additionalProperties": false,
+            "required": [
+              "identifier",
+              "item_number",
+              "item_specification_1",
+              "unit_of_sale",
+              "terms_of_sale_1",
+              "currency",
+              "country_of_origin"
+            ]
+          }
+        }
+      },
+      "additionalProperties": false,
+      "required": ["item"]
+    }
+  },
+  "additionalProperties": false,
+  "required": [
```

Comment:
Think this might need to be tweaked as things like `items`, `currency` etc is required?